🐙 source-retently: run up-to-date pipeline [2024-10-23]

airbyteio opened the pull request:
# Update source-retently

This PR was autogenerated by running `airbyte-ci connectors --name=source-retently up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Skipped

- Update versions of libraries in poetry.: Successful

- PATCH bump source-retently version to 0.2.24: Successful

- Build source-retently docker image for platform(s) linux/amd64, linux/arm64: Successful

- Get dependency updates: Failed

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=47108) <br/>All committers have signed the CLA.

airbyteio committed:
Update versions of libraries in poetry.
PATCH bump source-retently version to 0.2.24
Build source-retently docker image for platform(s) linux/amd64, linux/arm64
Create or update pull request on Airbyte repository
Add changelog entry
